src/linter/root.go

```diff
@@ -568,14 +569,19 @@ func (d *RootWalker) handleComment(c freefloating.String) {
 		return
 	}
 
+	if d.handledFiles == nil {
+		d.handledFiles = make(map[string]bool)
+	}
+
 	for _, ln := range phpdoc.Parse(d.ctx.phpdocTypeParser, str) {
 		if ln.Name() != "linter" {
 			continue
 		}
 
 		for _, p := range ln.(*phpdoc.RawCommentPart).Params {
-			if p == "disable" {
-				d.disabledFlag = true
+			if p == "disable" && !d.isDisabled && !d.handledFiles[d.ctx.st.CurrentFile] {
+				d.Report(n, LevelSecurity, "linterError", "You are not allowed to disable linter")
```

Comment:
I think it should be LevelInformation or something, but definitely not LevelSecurity. Security is for very specific kinds of checks that detect potential security vulnerabilities. Using a comment where it shouldn't be used is not a security issue. :)